python/lsst/daf/persistence/posixStorage.py

```diff
@@ -102,20 +85,23 @@ def getMapperClass(root):
             if os.path.exists(os.path.join(basePath, "_parent")):
```

Comment:
Note that `_parent` needs to be supported until repository configs are routinely persisted and the persisted configs are routinely used to construct repositories.